base/associative.jl

```diff
@@ -252,7 +252,7 @@ const hasha_seed = UInt === UInt64 ? 0x6d35bb51952d5539 : 0x952d5539
 function hash(a::Associative, h::UInt)
     h = hash(hasha_seed, h)
     for (k,v) in a
-        h $= hash(k, hash(v))
+        h = h ⊻ hash(k, hash(v))
```

Comment:
You could add `⊻=` in https://github.com/JuliaLang/julia/blob/5722529ac376012f845b8d12cfabff361148c5f4/src/julia-parser.scm#L11